README.md

```diff
@@ -20,21 +20,23 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
 
 ### Installation
 
-Make sure you have already installed the latest version of Node (at least version 4.2) and NPM (Node Package Manager) and then run:
-
-    npm install
+Make sure you have already installed the latest version of `Node` (at least version 6.10.3) and `NPM` (Node Package Manager) and then run:
 
+`npm install`
```

Comment:
I don't really see a reason for this change as from a visual appearance perspective they look the same in the end: `backticks` should be used for inline code, and

    four spaces indentation

for code blocks.